Automattic/wp-calypso
# Plans Grid: remove live chat and email support strings

Related to pau2Xa-5Vl-p2#comment-15472

## Proposed Changes

* This PR replaces references to live chat and email support from the plans grid and checkout.
<img width="1542" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/28f59c25-39b4-4f8a-9c55-2de153b08e22">

<img width="1605" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/15dc390a-b18e-494d-92bf-1bc8be33df1c">

<img width="1151" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/4482854b-301d-4c15-b218-5613891b4cfe">
<img width="1180" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/7c8dc32d-577e-466f-883c-cace921df4f6">




## Why are these changes being made?


* pau2Xa-5Ut-p2

## Testing Instructions


For the following scenarios, confirm that the Starter plan does not contain any references to support. References to support for the Explorer plan should have been replaced with 'Fast support from our expert team' and references for support for the Creator/Entrepreneur plan should have been replaced with 'Priority 24/7 support from our expert team'.
* `/start/plans` - pricing grid and comparison grid.
* `/setup/newsletter/plans`
* Trail map variants:
  * `/start/plans?flags=onboarding/trail-map-feature-grid-structure`
  * `/start/plans?flags=onboarding/trail-map-feature-grid-copy`
  * `/start/plans?flags=onboarding/trail-map-feature-grid`
* Add the Starter plan to the cart and confirm that the right sidebar checkout features include 'Fast support from our expert team'.
* Add the Explorer plan to the cart and confirm that the right sidebar checkout features include 'Priority 24/7 support from our expert team'.
## Pre-merge Checklist
